common/lib/protocol-definitions/src/summary.ts

```diff
@@ -7,13 +7,21 @@ export type SummaryObject = ISummaryTree | ISummaryBlob | ISummaryHandle | ISumm
 
 export type SummaryTree = ISummaryTree | ISummaryHandle;
 
+/**
+ * @deprecated - ISummaryAuthor has never been used anywhere and is unlikely to be used in the
```

Comment:
No dash when using the deprecated TSDoc tag. You should also get a lint error with this as it is. Please let me and @Josmithr know if you didn't, because it may be a bug in our lint config.

Reply:
this isn't causing a lint failure. the pr build passes

Reply:
It didn't complain about the dash. This is all the output:

```
@ghoshkaj ➜ .../FluidFramework/common/lib/protocol-definitions (depracate-ISummaryAuthor-ISummaryCommitter) $ npm run build

> @fluidframework/protocol-definitions@0.1028.3000 build /workspaces/FluidFramework/common/lib/protocol-definitions
> concurrently npm:build:compile npm:lint && npm run build:docs

[build:compile] 
[build:compile] > @fluidframework/protocol-definitions@0.1028.3000 build:compile /workspaces/FluidFramework/common/lib/protocol-definitions
[build:compile] > concurrently npm:typetests:gen npm:tsc npm:build:esnext
[build:compile] 
[lint] 
[lint] > @fluidframework/protocol-definitions@0.1028.3000 lint /workspaces/FluidFramework/common/lib/protocol-definitions
[lint] > npm run eslint
[lint] 
[lint] 
[lint] > @fluidframework/protocol-definitions@0.1028.3000 eslint /workspaces/FluidFramework/common/lib/protocol-definitions
[lint] > eslint --format stylish src
[lint] 
[build:compile] [typetests:gen] 
[build:compile] [typetests:gen] > @fluidframework/protocol-definitions@0.1028.3000 typetests:gen /workspaces/FluidFramework/common/lib/protocol-definitions
[build:compile] [typetests:gen] > fluid-type-validator -d .
[build:compile] [typetests:gen] 
[build:compile] [build:esnext] 
[build:compile] [build:esnext] > @fluidframework/protocol-definitions@0.1028.3000 build:esnext /workspaces/FluidFramework/common/lib/protocol-definitions
[build:compile] [build:esnext] > tsc --project ./tsconfig.esnext.json
[build:compile] [build:esnext] 
[build:compile] [tsc] 
[build:compile] [tsc] > @fluidframework/protocol-definitions@0.1028.3000 tsc /workspaces/FluidFramework/common/lib/protocol-definitions
[build:compile] [tsc] > tsc
[build:compile] [tsc] 
[build:compile] [typetests:gen] npm run typetests:gen exited with code 0
[build:compile] [tsc] npm run tsc exited with code 0
[build:compile] [build:esnext] npm run build:esnext exited with code 0
[build:compile] npm run build:compile exited with code 0
[lint] 
[lint] /workspaces/FluidFramework/common/lib/protocol-definitions/src/protocol.ts
[lint]   181:24  warning  Usage of "null" is deprecated except when describing legacy APIs; use "undefined" instead  @rushstack/no-new-null
[lint] 
[lint] /workspaces/FluidFramework/common/lib/protocol-definitions/src/sockets.ts
[lint]   28:21  warning  Usage of "null" is deprecated except when describing legacy APIs; use "undefined" instead  @rushstack/no-new-null
[lint] 
[lint] /workspaces/FluidFramework/common/lib/protocol-definitions/src/summary.ts
[lint]   35:1  warning  Unused eslint-disable directive (no problems were reported from '@typescript-eslint/no-namespace')
[lint] 
[lint] ✖ 3 problems (0 errors, 3 warnings)
[lint]   0 errors and 1 warning potentially fixable with the `--fix` option.
[lint] 
[lint] npm run lint exited with code 0

> @fluidframework/protocol-definitions@0.1028.3000 build:docs /workspaces/FluidFramework/common/lib/protocol-definitions
> api-extractor run --local --typescript-compiler-folder ./node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/


api-extractor 7.22.2  - https://api-extractor.com/

Using configuration from ./api-extractor.json
Analysis will use the bundled TypeScript version 4.5.5

API Extractor completed successfully
@ghoshkaj ➜ .../FluidFramework/common/lib/protocol-definitions (depracate-ISummaryAuthor-ISummaryCommitter) $ ```

Reply:
@tylerbutler Is there a specific rule you're thinking will fail? Just the tsdoc syntax mega-rule? My assumption would be that this would succeed, and the resulting bubbled-up documentation would just begin with "- ".

Reply:
@Josmithr I thought tsdoc/syntax had one, but I think I got it confused with the one for params:

```ts
  /**
   * The `@param` block should be followed by a parameter name and then a hyphen.
   */
  ParamTagMissingHyphen = 'tsdoc-param-tag-missing-hyphen',
```

https://github.com/microsoft/tsdoc/blob/b0499dace630ca562728f18faf434eda1c782650/tsdoc/src/parser/TSDocMessageId.ts#L143-L146

I wish the spec was consistent and had dashes everywhere or nowhere.

You're right that it just includes the hyphen in the docs, which looks silly (but nothing bad other than that):

<img width="494" alt="image" src="https://user-images.githubusercontent.com/19589/176976117-3293d909-a09c-400c-9e95-fa74bfddc975.png">